README.md

```diff
@@ -41,6 +41,99 @@ root.mainloop()
 ```
 
 
+## Tips and tricks
+Sun Valley is meant to be a simple ttk theme wrapped in a PyPI package for convenience. I don't want to add other dependencies, nor features that would be trivial to implement if the user wants them, using 3rd party modules. But this theme can be easily integrated with them.
+
+<details>
+  <summary><b>Set the theme to the system theme</b></summary>
+  
+  You can use [darkdetect](https://github.com/albertosottile/darkdetect) for that. Here's an example:
+  
+  ```python
+  import tkinter, sv_ttk, darkdetect
+  from tkinter import ttk
+
+  root = tkinter.Tk()
+
+  button = ttk.Button(root, text="Click me!", style="Accent.TButton")
+  button.pack()
+
+  sv_ttk.set_theme(darkdetect.theme())
+  root.mainloop()
+  ```
+
+  You just have to pass ```darkdetect.theme()``` to the ```sv_ttk.set_theme()``` function. It's that easy!
+</details>
+
+
+<details>
+  <summary><b>Dark Mode title bar on Windows</b></summary>
+
+  By default, sv_ttk doesn't change the title bar color on Windows when the theme is set to dark. But you can use [pywinstyles](https://github.com/Akascape/py-window-styles) to change the title bar color on Windows. Here's an example:
+  
+  ```python
+  import tkinter, sv_ttk
+  from tkinter import ttk
+
+  # Function to set the title bar color
+  def set_title_bar_color(root: tkinter.Tk | tkinter.Toplevel):
+      theme = sv_ttk.get_theme()
+
+      if get_windows_version() == 10:
+          import pywinstyles
+
+          if theme == "dark": pywinstyles.apply_style(root, "dark")
+          else: pywinstyles.apply_style(root, "normal")
+
+          # A hacky way to update the title bar's color on Windows 10 (it doesn't update instantly like on Windows 11)
+          root.wm_attributes("-alpha", 0.99)
+          root.wm_attributes("-alpha", 1)
+      elif get_windows_version() == 11:
+          import pywinstyles
+
+          # Set the title bar color to the background color on Windows 11 for better appearance
+          if theme == "dark": pywinstyles.change_header_color(root, "#1c1c1c")
+          elif theme == "light": pywinstyles.change_header_color(root, "#fafafa")
```

Comment:
Oh, yeah. It can be a bit confusing how these random colors get here. I think I will define them in sv-ttk with constants, but until then they can stay like this.